### PR TITLE
Fix/Update Documentation: Create and run Rasa bot

### PR DESCRIPTION
**Pull request by bandantonio**
Closes #1490

### CODE REVIEW

File: bots/create-and-run-a-bot/rasa-bot/README.md [hunk 1]
```diff
@@ -11,95 +11,93 @@ You will need Python knowledge only when creating custom actions in Rasa which c
...
+```
+git clone https://github.com/RocketChat/rasa-kick-starter
+cd rasa-kick-starter/
+```
 
-2. **configure the bot**
+**2. configure the bot**
 
-    1. open `rasa-kick-starter/bot_rasa` folder and update the `credentials.yml` file with Rasa bot's username, password, and Rocket.Chat URL:
+* open `rasa-kick-starter/bot_rasa` folder and update the `credentials.yml` file with Rasa bot's username, password, and Rocket.Chat URL:
```
Comment (reetp) [inline bots/create-and-run-a-bot/rasa-bot/README.md]: Should really be 'open _the_ rasa....'

File: bots/create-and-run-a-bot/rasa-bot/README.md [hunk 2]
```diff
@@ -11,95 +11,93 @@ You will need Python knowledge only when creating custom actions in Rasa which c
...
+$ docker-compose up bot_rasa
+Starting rasa-kick-starter_bot_rasa_1 ... done
+Attaching to rasa-kick-starter_bot_rasa_1
+bot_rasa_1            | 2019-10-31 21:23:24 INFO     root  - Starting Rasa server on http://localhost:5005
+```
 
-    ```
-    Hello from Rasa: 1.5.0a1
-    ```
+Open your browser and navigate to the `http://localhost:5005`. You should see the response from the running Rasa bot:
```
Comment (reetp) [inline bots/create-and-run-a-bot/rasa-bot/README.md]: no 'the' required

navigate to `http://localhost:5005`

File: bots/create-and-run-a-bot/rasa-bot/README.md [hunk 3]
```diff
@@ -11,95 +11,93 @@ You will need Python knowledge only when creating custom actions in Rasa which c
...
+* Post as: `<RASA USER NAME>`
 
-    **NOTE**: Make sure to replace the URL with a valid public URL obtained on the previous step using ngrok.
+**NOTE**: Make sure to replace the URL with a valid public URL obtained on the previous step using ngrok.
 
-    Click `Save changes` in the top right corner.
+Click `Save changes` in the top right corner.
 
-5. **talk to your bot**
+**5. talk to your bot**
```
Comment (reetp) [inline bots/create-and-run-a-bot/rasa-bot/README.md:100]: talkin _ing_ ?

File: bots/create-and-run-a-bot/rasa-bot/README.md [hunk 4]
```diff
@@ -11,95 +11,93 @@ You will need Python knowledge only when creating custom actions in Rasa which c
...
-    * Channel: `#general`
-    * URLs: `http://<ngrok_public_url>/webhooks/rocketchat/webhook`
-    * Post as: `<RASA USER NAME>`
+* Enabled: `True`
+* Channel: `#general`
+* URLs: `http://<ngrok_public_url>/webhooks/rocketchat/webhook`
+* Post as: `<RASA USER NAME>`
 
-    **NOTE**: Make sure to replace the URL with a valid public URL obtained on the previous step using ngrok.
+**NOTE**: Make sure to replace the URL with a valid public URL obtained on the previous step using ngrok.
```
Comment (reetp) [inline bots/create-and-run-a-bot/rasa-bot/README.md]: Make sure _you_

File: bots/create-and-run-a-bot/rasa-bot/README.md (hunk 3 above)
Comment (bandantonio) [reply bots/create-and-run-a-bot/rasa-bot/README.md:100]: no. all step are in infinitive: clone, configure, run, talk

File: bots/create-and-run-a-bot/rasa-bot/README.md (hunk 1 above)
Comment (bandantonio) [reply bots/create-and-run-a-bot/rasa-bot/README.md]: Fixed

File: bots/create-and-run-a-bot/rasa-bot/README.md (hunk 2 above)
Comment (bandantonio) [reply bots/create-and-run-a-bot/rasa-bot/README.md]: Fixed

File: bots/create-and-run-a-bot/rasa-bot/README.md (hunk 4 above)
Comment (bandantonio) [reply bots/create-and-run-a-bot/rasa-bot/README.md]: Fixed

File: bots/create-and-run-a-bot/rasa-bot/README.md [hunk 5]
```diff
@@ -93,7 +93,7 @@ Select `Message Sent` Event Trigger from the dropdown list. Complete the configu
 * URLs: `http://<ngrok_public_url>/webhooks/rocketchat/webhook`
 * Post as: `<RASA USER NAME>`
 
-**NOTE**: Make sure to replace the URL with a valid public URL obtained on the previous step using ngrok.
+**NOTE**: Make sure you replaced the URL with a valid public URL obtained on the previous step using ngrok.
```
Comment (reetp) [inline bots/create-and-run-a-bot/rasa-bot/README.md]: 'replace' not replaced.

'Make sure you replace the URL'

Comment (bandantonio) [reply bots/create-and-run-a-bot/rasa-bot/README.md]: Fixed

**Review by Sing-Li — APPROVED** [review]

**Review by MartinSchoeler — APPROVED** [review]
As for structurinf, Looks Good To Me!

File: bots/create-and-run-a-bot/rasa-bot/README.md [hunk 6]
```diff
@@ -0,0 +1,104 @@
+# Run a Rasa Bot
+
+Rasa is the leading open-source machine learning toolkit that lets developers expand bots beyond answering simple questions
+with minimal training data. At the core, Rasa bot has a machine learning model which trained on example conversations.
```
Comment (reetp) [inline bots/create-and-run-a-bot/rasa-bot/README.md:4]: This was the original:
The bots are based on a machine learning model trained on example conversations.

One of these two:
Rasa bot has a machine learning model which "has been" trained on trained on example conversations
Rasa bot has a machine learning model trained on example conversations

File: bots/create-and-run-a-bot/rasa-bot/README.md [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+```
+
+Open your browser and navigate to `http://localhost:5005`. You should see the response from the running Rasa bot:
+
+```
+Hello from Rasa: 1.5.0a1
+```
+
+If you have Rocket.Chat running on the same machine, the bot's URL is `http://bot_rasa:5005`.
+This guide uses a remote Rocket.Chat instance, so it is necessary to get a public URL for the Rasa bot
```
Comment (reetp) [inline bots/create-and-run-a-bot/rasa-bot/README.md:62]: Is there no guide if running on the SAME server as Rocket.Chat?

Comment (reetp) [reply bots/create-and-run-a-bot/rasa-bot/README.md:62]: Thinking 
'Rasa bot with Local Instance'
'Rasa bot with Remote Instance'

**Pull request merged**